docker-compose.yaml

```diff
@@ -1,9 +1,9 @@
 version: '3'
 services:
   neo4j:
-    image: neo4j:3.5.0
+    image: neo4j:4.1.5
     environment:
-      - NEO4J_dbms_logs_query_enabled=true
+      - NEO4J_dbms_logs_query_enabled=VERBOSE
```

Comment:
Slightly confused now with these settings as this v4.1 database can now be accessed by a v4 driver without the need for me to be explicit about the use of a certificate!!

Reply:
You're using `neo4j` protocol not `neo4j+ssc` https://github.com/Financial-Times/treecreeper/pull/331/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R20

Reply:
Yes, I've now realised that at well, silly me.   However, this PR will now focus on the functionality delivered by the neo4j driver (e.g. CYPHER) not the security.   The security will be tested via a similar PR for Biz Ops API